src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.java

```diff
@@ -95,6 +95,7 @@ public UnifiedCompactionStrategy(ColumnFamilyStore cfs, Map<String, String> opti
         super(cfs, options);
         this.controller = controller;
         estimatedRemainingTasks = 0;
+        lastExpiredCheck = Clock.Global.currentTimeMillis();
```

Comment:
Without this an expiration check was carried out almost immediately, which broke some tests.